test/unit/pages/ClinicPatients.test.js

```diff
@@ -577,7 +581,6 @@ describe('ClinicPatients', () => {
         </Provider>
       );
 
-      wrapper.find('#patients-view-toggle').hostNodes().simulate('click');
```

Comment:
Hope you're okay with this. Because the state is defined in the redux store, and we're testing all of the functionality where the patient list is visible, I need to default the redux store value of "Show Patients" to true here, which is not actually the default state of the page.

I'm not aware of any capability we have to actually hook up a functioning redux store inside the test such that we can actually default "Show Patients" to false and then add in the simulated interaction.

So my thought is that we test all of this functionality while the redux state is set to true, and then also add tests to make sure the the redux state will get changed by the inputs.

Reply:
This approach seems reasonable to me. The alternative, as I see it, would be to just use the _actual_ redux store with our reducers wrapped to capture actions on the way through, but that really starts to fall outside the scope of "unit" tests.